src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const rightElementsOfString = sourceString.split(';').map((property) =>
```

Comment:
wordy naming gives us much more context than `object` or sth, so good job with it 🚀 
But if you can make it shorter - you should avoid unnecessary wordiness
```suggestion
  const validProperties = sourceString.split(';').map((property) =>
```